### PR TITLE
chore(TES): adjust expired credential response

### PR DESCRIPTION
**Pull request by jcosentino11**
**Issue #, if available:**

**Description of changes:**

Adjust the response TES gives upon encountering expired credentials, such that the response does not contain the expired credentials.  This prevents expired credentials from being logged [later on](https://github.com/aws-greengrass/aws-greengrass-nucleus/blob/4bb96750c21bc02de3fd3fd6182afb8ecba41b46/src/main/java/com/aws/greengrass/tes/CredentialRequestHandler.java#L357).

**Why is this change necessary:**

**How was this change tested:**
- [x] Updated or added new unit tests.
- [ ] Updated or added new integration tests.
- [ ] Updated or added new end-to-end tests.
- [ ] If my code makes a remote network call, it was tested with a proxy.

**Any additional information or context required to review the change:**

**Documentation Checklist:**
 - [ ] Updated the README if applicable.

**Compatibility Checklist:**
- [x] I confirm that the change is backwards compatible.
- [ ] Any modification or deletion of public interfaces does not impact other plugin components.
- [ ] For external library version updates, I have reviewed its change logs and Nucleus does not consume 
  any deprecated method or type.

Refer to [Compatibility Guidelines](/COMPATIBILITY.md) for more information.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: src/main/java/com/aws/greengrass/tes/CredentialRequestHandler.java [hunk 1]
```diff
@@ -226,7 +226,7 @@ private byte[] getCredentialsBypassCache() {
                     Instant expiry = Instant.parse(expiryString);
 
                     if (expiry.isBefore(Instant.now(clock))) {
-                        String responseString = "TES responded with expired credentials: " + credentials;
```
Comment (MikeDombo) [inline src/main/java/com/aws/greengrass/tes/CredentialRequestHandler.java:229]: Let's keep this somewhat useful. Add in the expiration date to the message at least.

**Comment by MikeDombo** [comment]
You have a PMD failure

File: src/main/java/com/aws/greengrass/tes/CredentialRequestHandler.java (hunk 1 above)
Comment (jcosentino11) [reply src/main/java/com/aws/greengrass/tes/CredentialRequestHandler.java:229]: added

**Review by MikeDombo — APPROVED** [review]

**Comment by shaguptashaikh** [comment]
Curious what causes TES to return expired credentials to begin with?

**Review by shaguptashaikh — APPROVED** [review]

**Review by shaguptashaikh — DISMISSED** [review]

File: src/main/java/com/aws/greengrass/tes/CredentialRequestHandler.java [hunk 2]
```diff
@@ -354,7 +354,6 @@ private AwsCredentials getCredentialsFromByte(byte[] data) {
                             credentials.get(SESSION_TOKEN_DOWNSTREAM_STR));
         } catch (IOException e) {
             LOGGER.atError().kv(IOT_CRED_PATH_KEY, iotCredentialsPath)
-                    .kv("credentialData", new String(data, StandardCharsets.UTF_8))
```
Comment (MikeDombo) [inline src/main/java/com/aws/greengrass/tes/CredentialRequestHandler.java:357]: do not remove this, this includes very very very useful information

**Review by shaguptashaikh — APPROVED** [review]

**Review by MikeDombo — APPROVED** [review]

**Review by indougnito — APPROVED** [review]

**Comment by jcosentino11** [comment]
Can someone merge this, I don't have permissions :) @MikeDombo @shaguptashaikh @indougnito

**Pull request merged**